docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -217,28 +279,54 @@ Go over the output tensors and process the inference results.
 
 @endsphinxtab
 
+@sphinxtab{C}
+
+@snippet docs/snippets/src/main.c part6
+
+@endsphinxtab
+
 @endsphinxtabset
 
-## Step 7. Link and Build Your Application with OpenVINO™ Runtime (example)
+## Step 7. Release the allocated objects (only for C)
 
-This step may differ for different projects. In this example, a C++ application is used, together with CMake for project configuration.
+To avoid memory leak, applications developed with C API need to release the allocated objects in order.
+
+@sphinxtabset
+
+@sphinxtab{C}
+
+@snippet docs/snippets/src/main.c part8
+
+@endsphinxtab
+
+@endsphinxtabset
+
+## Step 8. Link and Build Your Application with OpenVINO™ Runtime (example)
+
+This step may differ for different projects. In this example, a C++ & C application is used, together with CMake for project configuration.
 
 For details on additional CMake build options, refer to the [CMake page](https://cmake.org/cmake/help/latest/manual/cmake.1.html#manual:cmake(1)).
 
-### Create a structure for the project:
-   ``` sh
-   project/
-       ├── CMakeLists.txt  - CMake file to build
-       ├── ...             - Additional folders like includes/
-       └── src/            - source folder
-           └── main.cpp
-   build/                  - build directory
-       ...      
-   ```
+@sphinxtabset
+
+@sphinxtab{C++}
+
+@snippet docs/snippets/src/main.cpp part7
 
-### Include OpenVINO™ Runtime libraries in `project/CMakeLists.txt`
+@snippet snippets/CMakeLists.txt cmake:integration_example_cpp
+
+@endsphinxtab
+
+@sphinxtab{C}
```

Comment:
I propose to add introduction before, something like: "Create CMake script:"
Because in other case we have 2 different sections without any comment:
<img width="937" alt="Screenshot 2022-10-11 at 07 42 19" src="https://user-images.githubusercontent.com/4737252/194992241-a4f9dfb3-3679-4aea-be75-4baf820d64d8.png">